src/Wallabag/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -208,6 +208,8 @@ entry:
         url_label: 'Url'
         is_public_label: 'Public'
         save_label: 'Save'
+    public:
+        shared_by_wallabag: "This article has been shared by <a href='https://www.wallabag.org'>wallabag</a>"
```

Comment:
It's an open question I don't know if it's better to put a link to the _brand_ or to the user instance.